@attr(tags=['simulator', 'basic', 'advanced'])
def test_05_create_aff_grp_same_name_diff_acc(self):
    self.user = Account.create(self.api_client, self.services['new_account'], domainid=self.domain.id)
    self.cleanup.append(self.user)
    self.create_aff_grp(aff_grp=self.services['host_anti_affinity_0'], acc=self.user.name, domainid=self.domain.id)
    try:
        self.create_aff_grp(aff_grp=self.services['host_anti_affinity_0'])
    except Exception:
        self.debug('Error: Creating affinity group with same name from different account failed.')
    AffinityGroup.delete(self.api_client, name=self.aff_grp.name, account=self.user.name, domainid=self.domain.id)
    self.debug(('Deleted Affinity Group: %s' % self.aff_grp.name))
